test/integration/checkout_int_test.go

```diff
@@ -370,6 +370,13 @@ func (suite *CheckoutIntegrationTestSuite) TestCheckoutFromArchive() {
 	cp.Expect("All dependencies have been installed and verified")
 	cp.Expect("Checked out project ActiveState-CLI/AlmostEmpty")
 	cp.ExpectExitCode(0)
+
+	// Verify the zlib runtime files exist.
+	proj, err := project.FromPath(filepath.Join(ts.Dirs.Work, "AlmostEmpty"))
+	suite.Require().NoError(err)
+	cachePath := filepath.Join(ts.Dirs.Cache, runtime_helpers.DirNameFromProjectDir(proj.Dir()))
+	zlibH := filepath.Join(cachePath, "usr", "include", "zlib.h")
```

Comment:
I'm pretty sure this will be different on mac/windows.

Reply:
Actually, it can be whatever I say it is since I produced the test archives :) But seriously, I untarred the ingredients and looked for a common one that had the same path and used that. The tests on all three platforms are passing, so it's all good.

Reply:
Oh right, I forgot about that detail :p Ok we'll worry about this once we can test with proper payloads.